api/src/pcapi/settings.py

```diff
@@ -98,6 +98,9 @@
 # USERS
 MAX_FAVORITES = int(os.environ.get("MAX_FAVORITES", 100))  # 0 is unlimited
 MAX_API_KEY_PER_OFFERER = int(os.environ.get("MAX_API_KEY_PER_OFFERER", 5))
+USE_FAST_AND_INSECURE_PASSWORD_HASHING_ALGORITHM = bool(
+    int(os.environ.get("USE_FAST_AND_INSECURE_PASSWORD_HASHING_ALGORITHM", IS_DEV))
+)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏Dommage de retirer `IS_DEV` mais le garder ici :confused: 

```suggestion
USE_FAST_AND_INSECURE_PASSWORD_HASHING_ALGORITHM = bool(
    int(os.environ.get("USE_FAST_AND_INSECURE_PASSWORD_HASHING_ALGORITHM", False))
)
```

Reply:
c'est juste, j'ai mis à jour le commit